src/views/studio/studio-curators.jsx

```diff
@@ -1,16 +1,77 @@
-import React from 'react';
+import React, {useEffect, useCallback} from 'react';
+import PropTypes from 'prop-types';
 import {useParams} from 'react-router-dom';
+import {connect} from 'react-redux';
+
+import {curators, managers} from './lib/redux-modules';
+import {curatorFetcher, managerFetcher} from './lib/fetchers';
+import Debug from './debug.jsx';
 
 const StudioCurators = () => {
     const {studioId} = useParams();
-
     return (
         <div>
-            <h2>Curators</h2>
-            <p>Studio {studioId}</p>
-
+            <h3>Managers</h3>
+            <ManagerList studioId={studioId} />
+            <hr />
+            <h3>Curators</h3>
+            <CuratorList studioId={studioId} />
         </div>
     );
 };
 
+const MemberList = ({studioId, items, error, loading, moreToLoad, onLoadMore}) => {
+    useEffect(() => {
+        if (studioId && items.length === 0) onLoadMore(studioId, 0);
+    }, [studioId]);
+    
+    const handleLoadMore = useCallback(() => onLoadMore(studioId, items.length), [studioId, items.length]);
```

Comment:
The immediate reason is that our linter wont let us pass anonymous functions to react components. That rule feels a bit strict, especially because function components don't do any "shouldComponentUpdate" checking at all by default, so changing the reference on a callback function isn't making things any worse. But if we were using extensive use of `shouldComponentUpdate` or `React.memo`, the `useCallback` hook is a way to keep a stable reference to a function. Since the callback uses data from inside of the render function, it needs to change reference any time the data used internally changes. So `useCallback` here takes a list of data the callback depends on, and will regenerate a new function closed over the new data any time it changes.

It's not an optimization I would have made necessarily, but that linter is strict!